Packs/FeedThreatConnect/README.md

```diff
@@ -1,3 +1,7 @@
-The Threat Connect feed provides access to published IOCs that contain known indicators. You can configure the number of indicators to be returned. The default is 500.
+The Threat Connect feed provides access to published IOCs that contain known indicators. You can configure the number of indicators to be returned. The default is 50.
```

Comment:
```suggestion
The Threat Connect feed provides access to published IOCs that contain known indicators.
```
Actually there is no way of configuring the number of indicators that returns from the integration.